#request
---
#response
bool emergencyStopped
ArmState state
